src/Tests.AzureStorage2/Tests.AzureStorage2.csproj

```
@@ -18,15 +21,15 @@
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.40.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.6.0" />
-    <PackageReference Include="Nunit" Version="3.14.0" />
+    <PackageReference Include="Nunit" Version="4.2.1" />
```

```suggestion
    <PackageReference Include="NUnit" Version="4.2.1" />
```

Do we want to align the package name everywhere?